source/operating-a-service/incident-playbook.html.md

```diff
@@ -0,0 +1,87 @@
+    (responsible for monitoring Zendesk and alerting the team to any changes in the queues and severity of experience of users).
+
+### 2. Triage the incident (all incident leads)
+
+The incident leads should [triage the incident](/operating-a-service/how-to-categorise-technical-incidents.html) (P1, P2, P3).
+
+### 3. Create an incident Slack channel and inform the stakeholders (comms lead)
+
+1. Initiate the Slack IncidentBot by typing `/incident` in the message box on #twd_git_bat channel or the #tra_digital channel (as appropriate), and hit Enter.
+2. Complete the details in the IncidentBot template, and press Enter, which will automatically create a dedicated slack channel for the incident.
+3. Determine who needs to be contacted, based on the incident priority and affected services, using [the incident contact lists](https://docs.google.com/document/d/1E3sL-Om_NPHWHdYdLykVuiWux_6AmR5pn5KjIkQYHaI/edit#bookmark=id.djbosiwhbjjy).
+4. @tag the appropriate people (from the tables linked below) in the dedicated channel:
+
+### 4. Provide a service update to users outside DfE (comms lead)
+
+The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). During an incident, the comms lead needs to explain what’s happening to users outside DfE. The comms lead will need a GitHub account to do this, or delegate updates to a colleague who has one.
+
+The updates are managed via GitHub Actions and issues on the [teacher-services-upptime repository on GitHub](https://github.com/DFE-Digital/teacher-services-upptime). If a service’s automatic health check is failing continuously, an issue will be created within 5 minutes of the failure occurring and the dashboard will start reporting a service issue.
+
+To update the dashboard:
+
+1. Navigate to the appropriate incident issue on the [GitHub issues page](https://github.com/DFE-Digital/teacher-services-upptime/issues)
+2. Add a comment to the issue
+
+### 5. Start the incident report (any incident lead)
+
+Create the incident report using the template in Google Drive:
+
+- Create a running [Incident Report using this template](https://docs.google.com/document/d/1HwKCPafnluOIhIAWbSD91zxt7w3q4FGDIVKS3d_SDFA/edit?usp=sharing)
+- Rename the created file to include today’s date and save as a new file in the [Incident reports folder](https://drive.google.com/drive/folders/12uWIF4beypUpEjejTRcKtV2PFadT5met)
+
+### 6. Decide whether to contact users about an incident (support lead)
+
+Contact your users if:
+
+* The incident will negatively impact them for a prolonged period of time
+* It can pre-empt a high volume of support tickets
+
+Informing users about incidents is generally considered best practice, but should be decided on a case by case basis with the product and service managers.
+
+## While the incident is in progress
+
+Keep all conversations and status updates about the incident on the dedicated Slack incident channel.
```

Comment:
There is also an `/update` command which will allow the requester to update any one, or all, of the following fields:

- Description
- Priority
- Leads

Is this worth mentioning?

Reply:
Yup – I'll update this section.

Reply:
Updated in e2e906c599514d001b11fec0c74af39b2963f489